### ISSUE TITLE
Feature Request - encumbrance expirations and cleanup

### ISSUE THREAD

**Issue by ccicero** [ArctosDB/arctos#5028]
The Code Table Committee met today and spent some time talking about encumbrances and adding definitions. @Jegelewicz is filing a separate issue about encumbrance actions, but the issue of expirations came up. There are encumbrances that have expired, encumbrances that will be expiring even though the person who added the encumbrance made it clear that they should never expire, encumbrances by people who are no longer part of Arctos (e.g., Charles Dardia at CUMV), encumbrances with no associated records, etc.

Request is as follows:

1) Conduct a review of encumbrance and delete ones that don't have any associated records or that were created by people that are no longer with Arctos; if appropriate (e.g., institution is still part of Arctos even if staff has changed), change the encumbrance so that it's affiliated with a current Arctos operator/collection contact so that notifications are received by the appropriate person.

2) Allow the option of creating an encumbrance with no expiration date, or at least with one that's 50 (?) years. The default should **not** be that when an encumbrance expires, data automatically become public. And if there is a currently timeline of 5 years (?). that is not long enough. This is especially true for sensitive data (agents, coordinates, NAGPRA, etc.).  Although notifications are sent, we can't rely on people to pay attention and update their encumbrance. These 'indefinite' encumbrances should have to be manually changed to unencumber them, vs defaulting to unencumbered when the encumbrance expires (many are set to expire 2026-03-01).

3) Add something in the Newsletter at the end of the year, or start of the next year, that asks people to review any encumbrances for their collection. This could/should be an annual thing.

**Comment by dustymc** [comment]
>Allow the option of creating an encumbrance with no expiration date

There is a very long conversation on this (probably in google), the current behavior is from people forever-encumbering things that don't need to be encumbered at all. I'm not suggesting the current setup is the final answer, but we didn't get there arbitrarily.

Maybe encumbrances should have a financial cost??

**Comment by Jegelewicz** [comment]
Bring to AWG so we can make a social compact!

**Comment by AJLinn** [comment]
> Maybe encumbrances should have a financial cost??

I don't see this as a responsible solution. Cultural collections encumber insurance values as a security issue, because we need to have a value documented for insurance purposes, BUT we certainly don't want to broadcast that an item sitting in our gallery outside of an exhibit case is worth $500,000...

We also encumber culturally sensitive information that tribes have shared, which they want included with our object records but don't want that info broadcast to the general public.

These are institutionally and socially responsible actions and I don't think we should be charged extra for doing our jobs ethically (when we're already paying probably more than anyone else...)

**Comment by Jegelewicz** [comment]
Agree 100% with Angie. MOST encumbrances are the result of an out side request or legal requirement. Asking institutions to pay for that is not going to help them manage the objects or data in their care.

**Comment by campmlc** [comment]
Agree. This is the downside to online data management, and we should
support it because we value all the benefits. If we don't provide these
tools, collections needing them will be forced to use a system that allows
them hide their data locally and only publish via aggregators.

**Comment by dustymc** [comment]
> as a security issue

Seems a good candidate for '[internal by decree](https://github.com/ArctosDB/arctos/issues/3536#issuecomment-1405371785)' (should that turn into something).

ANYWAY - I'm just struggling to find SOMETHING that will result in things that don't need to be encumbered not remaining encumbered indefinitely because nobody reads notifications. Surely there's a workable balance out there somewhere, I don't think we've yet found it.

**Comment by Jegelewicz** [comment]
> Allow the option of creating an encumbrance with no expiration date, or at least with one that's 50 (?) years. The default should not be that when an encumbrance expires, data automatically become public. And if there is a currently timeline of 5 years (?). that is not long enough. This is especially true for sensitive data (agents, coordinates, NAGPRA, etc.). Although notifications are sent, we can't rely on people to pay attention and update their encumbrance. These 'indefinite' encumbrances should have to be manually changed to unencumber them, vs defaulting to unencumbered when the encumbrance expires (many are set to expire 2026-03-01).

AWG agreed today that
1. An expiration date should NOT be required for encumbrances
2. An encumbrance WITHOUT an expiration date will NOT automatically expire
3. The encumbrance expiration date can be set for ANY date in the future
